tests/Resources/ti.geolocation.addontest.js

```diff
@@ -0,0 +1,62 @@
+/*
+ * Appcelerator Titanium Mobile
+ * Copyright (c) 2011-Present by Appcelerator, Inc. All Rights Reserved.
+ * Licensed under the terms of the Apache Public License
+ * Please see the LICENSE included with this distribution for details.
+ */
+/* eslint-env mocha */
+/* global Ti */
+/* eslint no-unused-expressions: "off" */
+'use strict';
+var should = require('./utilities/assertions');
+
+describe.android('Titanium.Geolocation', function () {
+	// Methods
+
+	it('async #forwardGeocoder()', function (finish) {
+		this.timeout(6e4); // 60 sec
+
+		should(Ti.Geolocation.forwardGeocoder).be.a.Function;
+		Ti.Geolocation.forwardGeocoder('440 N Bernardo Ave, Mountain View')
+			.then(function (data) {
+				should(data).have.property('success').which.is.a.Boolean;
+				should(data.success).be.eql(true);
+				should(data).have.property('code').which.is.a.Number;
+				should(data.code).be.eql(0);
+				should(data.latitude).be.approximately(37.387, 0.002); // iOS gives: 37.38605, Windows does 37.3883645
+				should(data.longitude).be.approximately(-122.065, 0.02); // WIndows gives: -122.0512682, iOS gives -122.08385
+				finish();
+			})
+			.catch(finish);
+	});
+
+	// FIXME The address object is different from platform to platform! https://jira.appcelerator.org/browse/TIMOB-23496
+	it('async #reverseGeocoder()', function (finish) {
+		this.timeout(6e4); // 60 sec
+
+		should(Ti.Geolocation.reverseGeocoder).be.a.Function;
+		Ti.Geolocation.reverseGeocoder(37.3883645, -122.0512682)
+			.then(function (data) {
```

Comment:
<!--
  0 failure: 
  1 warning:  tests/Resources/t...
  
  
  DangerID: danger-id-default;
  File: tests/Resources/ti.geolocation.addontest.js;
  Line: 39;
-->

- :warning: tests/Resources/ti.geolocation.addontest.js line 39 – Each then() should return a value or throw (promise/always-return)